src/lib/components/ManagePanel.js

```diff
@@ -12,12 +11,12 @@ export default class ManagePanel extends React.Component {
     this.state = {
       datepickerActive: false,
       editedItem: null,
-      dateChoice: moment()
+      dateChoice: this.props.moment()
```

Comment:
Could just use `props` here, since it's passed into the constructor. But, guessing this was a find/replace maybe